frontend/package.json

```diff
@@ -45,23 +45,23 @@
     "@octokit/rest": "19.0.3",
     "@octokit/types": "6.40.0",
     "@openshift-assisted/locales": "2.9.16-cim",
-    "@openshift-assisted/ui-lib": "2.9.16-cim",
+    "@openshift-assisted/ui-lib": "^2.9.16-cim",
```

Comment:
@robdolares For future reference, you can use the `--save-exact` option to avoid having `npm` add the `^`.